l10n_it_fatturapa_out/wizard/__init__.py

```diff
@@ -7,6 +7,6 @@
 # there are multiple statements that should be guarded
 # in wizard_export_fatturapa, so we guard the entire file
 try:
-    from . import wizard_export_fatturapa
+    from . import wizard_export_fatturapa_regenerate  # noqa
```

Comment:
Visto che questo non è un semplice spostamento, forse lo terrei su un commit separato insieme alle relative modifiche. Che ne pensi @eLBati ?

Reply:
@primes2h in realtà è solo uno spostamento; anche questo dipende dallo stato. Ho semplicemente modificato il nome del file perché era già presente.